### PR TITLE
Rewrite the lib to ES5

### PR DESCRIPTION
**Pull request by Avaq**

### CODE REVIEW

File: inspect-f.js [hunk 1]
```diff
@@ -31,20 +31,20 @@
...
 
   function guessIndentation(lines){
-    const filtered = lines.filter(x => RCODE.test(x));
-    const paddings = filtered.map(getPadding);
-    const depth = paddings.reduce((a, b) => Math.min(a, b), Infinity);
-    const tabsize = paddings.map(x => x - depth).find(x => x > 1) || 2;
+    var filtered = lines.filter(x => RCODE.test(x));
+    var paddings = filtered.map(getPadding);
+    var depth = paddings.reduce((a, b) => Math.min(a, b), Infinity);
+    var tabsize = paddings.map(x => x - depth).find(x => x > 1) || 2;
```
Comment (lithin) [inline inspect-f.js]: arrow functions need to be converted as well

Comment (Avaq) [inline inspect-f.js]: That's odd. The arrow functions in the unit tests were giving me syntax errors in ESLint.

Comment (Avaq) [inline inspect-f.js]: Ah. I see. The `npm run lint` command tries to run on `fluture.js` >.<

**Comment by Avaq** [comment]
I fixed the linting script in `master` and rebased, which allowed me to fix the ES6 related syntax errors.

**Comment by lithin** [comment]
👍 good to go I think :)

**Pull request merged**